Data ingestion

earny-joe opened the pull request:
This pull request involves the following updates:

- updated `bleach` to address a security vulnerability
- added `.gitignore`, removed `.gitkeep`s
- Jupyter notebooks detailing potential text preprocessing pipeline
- cleaned up `experiments` folder, moved notebooks with older work to sub-folder `playground_nbs`
- Initial EDA Jupyter Notebook assessing data gathered from Panacea Lab's COVID-19 Twitter repo

rkingery commented:
Some comments:
- Your EDA looks good. Seems like things are more or less okay statistically speaking. 
- I'd come up with a better convention of naming your notebooks. The number convention I find confusing. Maybe just giving them semantic names instead of trying to maintain order is a good idea. Also, no point in tagging your name to the notebook names, as it's generally obvious who created them (github tracks it). Not sure who started this notebook naming convention but I don't care much for it.
- Would be interesting to do a semantic analysis as well in the EDA later on, e.g. by using embeddings to see which terms / hashtags/ etc are close to covid-19 related words as well as phrases associated with misinformation. I'd be particularly interested in seeing how some of these news agencies cluster together in an embedding space and if an info/misinfo pattern can be gleaned from that.
- The file you call `fundamentals.py` is really a utilities file (a file that you use over and over to load common functions in your notebooks/scripts). A more common naming convention for this file is `utils.py`, which you may wanna use instead.
- Not sure if you have them yet, but any configs that you find yourself using over and over again (e.g. paths, urls, etc), consider putting in a `configs.py` file and loading them. That way when you change them you only have to do so in one place.
- You're getting better at using the markdown functionality in your notebooks, but try to work on it some more. Get to the point where you're making your notebooks read like a story with code/results inside.

earny-joe commented:
- cleaned up Jupyter notebook naming convention to be more user-friendly
- changed `fundamentals.py` to `utils.py`
- added `experiments/playground_data` (i.e. a folder containing data to be more easily accessed for analysis) to `.gitignore`
